spec/glossary_v3/g_common.yaml

```diff
@@ -45,6 +45,8 @@ components:
   schemas:
     GetGlossaryEntriesByFilterCommandPTO:
       type: object
+      required:
+       - entryState
```

Comment:
required fields of the `object` described in the separate array field `required`